requirements.txt

```diff
@@ -1,4 +1,4 @@
-beautifulsoup4==4.9.1
+dnspython==1.16.0
```

Comment:
Is this the latest working version? It's 6 years old.

Reply:
Hey @bb-splunk,

The latest version of threatstream 3.5.3  uses `dnspython 2.6.1`.  'dnspython 2.6.1' has some issues in the dns. query. The latest changes are not yet released by the 'dnspython' library contributors. Therefore we have reverted to the previous version of 'dnspython' which is 1.16.0, which the customer is currently using.  For more info on this issue please check SOARHELP-3413 and PAPP-34172.

Reply:
Yeah I know, but isn't there a later version that doesn't have the bug?

Reply:
In future, when the package will be upgraded on pip, we'll release a new version of threatstream with new dnspython package. We have reverted back to 1.16.0 for the meantime. `1.16.0` was the last version which was used in the app and there were no issue related to dns.

Reply:
Ok then, is there already a ticket for a follow-up?

Reply:
Yes, you can check out PAPP-34327.